packages/expo-cli/src/commands/build-native/iOSBuilder.ts

```diff
@@ -67,6 +74,63 @@ class iOSBuilder implements Builder {
     this.credentials = await provider.getCredentialsAsync(credentialsSource);
   }
 
+  public async configureProjectAsync(): Promise<void> {
+    if (!this.credentials || !this.bundleIdentifier) {
+      throw new Error('Call ensureCredentialsAsync first!');
```

Comment:
missing credentials field is a normal for some builds
if you don't want to handle that case for now just return from function